web/src/regression/apiDriver.ts

```diff
@@ -0,0 +1,97 @@
+/**
+ * Provides convenience functions for interacting with the Sourcegraph API from tests.
+ */
+
+import { gql } from '../../../shared/src/graphql/graphql'
+import * as GQL from '../../../shared/src/graphql/schema'
+import { GraphQLClient } from './api'
+import { retry } from '../../../shared/src/e2e/e2e-test-utils'
+
+/**
+ * Block until all repositories in the list exist
+ */
+export async function waitForRepos(gqlClient: GraphQLClient, ensureRepos: string[]): Promise<void> {
+    for (const repoName of ensureRepos) {
+        await retry(async () => {
+            const res = await gqlClient
+                .queryGraphQL(
+                    gql`
+                        query ResolveRev($repoName: String!, $rev: String!) {
```

Comment:
```suggestion
                        query ResolveRev($repoName: String!) {
```

`$rev` is now unused.